doc/mgr/balancer.rst

```diff
@@ -0,0 +1,188 @@
+balancer plugin
+================
+
+Typically, if you say, the system is 70% filled, then, all the OSDs,
+should be 70% filled (roughly). It shouldn’t be the case that, a particular
+OSD is 90% filled whereas another one is 20% filled. If such a case exist,
+then we say there is an uneven distribution. Uneven distribution may arise
+out of a number of reasons, including improper distribution of PGs among the
+OSDs. Even if the PGs, were balanced properly, with respect to the weights
+of the OSDs, there is still chance of uneven distribution, because the number
+of objects assigned to different PGs are not in required proportion. And even
+if they were, the sizes of each object might vary.
+
+The balancer plugin will automatically optimize the pg (or objects or byte-level)
+distribution.  You need to just turn it on and it will slowly and continuously
+optimize the layout without the user having to think about it.
+
+While it is recommended that the balancer module be best left to run auto-magically,
+it might interest certain users to turn off the automatic optimization, and instead
+create optimization plans themselves, and execute them, whenever they feel the need
+to do so. While there is no fixed way to determine when to execute the rebalance
```

Comment:
@tchaikov By this, we will be able to tell how uneven the distribution in the cluster is.